sleepyinstant/src/lib.rs

```diff
@@ -0,0 +1,124 @@
+//! Attempts to provide the same functionality as std::time::Instant, except it
+//! uses a timer which accounts for time when the system is asleep
+use std::time::Duration;
+
+#[cfg(target_os = "windows")]
+mod inner {
+    pub use std::time::Instant;
+}
+
+#[cfg(unix)]
+#[path = "unix.rs"]
+mod inner;
```

Comment:
I'm not a fan of explicitly setting paths like this, as it makes it harder to track things.